### DISCUSSION TITLE
uninstallation?

### DISCUSSION THREAD

**Discussion by ghost** [RainbowMiner/RainbowMiner#1691] [Q&A] [answered]
rainbowminer never worked for me, and i'd like to remove the systemd service it installed that continues to run every boot. unfortunately, the service isn't actually named rainbowminer. what's it named so i can remove it?

**Comment by RainbowMiner** [comment, marked as answer]
Oh, that is sad. The systemd service is `ocdaemon`

**Comment by ghost** [comment]
yeah, i think my drivers are weird. great software though, thanks for the work and the answer!
